boards/arm/mimxrt1064_evk/mimxrt1064_evk.dts

```diff
@@ -250,8 +250,14 @@ zephyr_udc0: &usb1 {
 
 &lpspi1 {
 	status = "okay";
+	/* DMA channels 0 and 1, muxed to LPSPI1 RX and TX */
+	dmas = <&edma0 0 13>, <&edma0 1 14>;
+	dma-names = "rx", "tx";
 };
 
 &lpspi3 {
 	status = "okay";
+	/* DMA channels 2 and 3, muxed to LPSPI3 RX and TX */
+	dmas = <&edma0 2 15>, <&edma0 3 16>;
+	dma-names = "rx", "tx";
```

Comment:
Should changes to this file move the commit related to changes to the `boards` folder

Reply:
Fixed, thank you